core/tests/test_transform.h

```diff
@@ -0,0 +1,252 @@
+//
+//  test_transform.h
+//  engine-unit-tests
+//
+//  Created by Xavier Legland on 10/05/2022.
+//
+
+#pragma once
+
+#include "scene.h"
+#include "transform.h"
+
+void test_transform_rotation_position(void) {
+    Transform* t = transform_make_default();
+    TEST_ASSERT(t != NULL);
+
+    {
+        Quaternion* q = quaternion_new(1.0f, 2.0f, 3.0f, 0.5f, false);
+        transform_set_rotation(t, q);
```

Comment:
I think `Transform* t` declared line `14`, should be inside this block. It shouldn't be used for multiple tests.
Don't forget to move the `transform_release(t);` inside this block as well.